src/bp-core/classes/class-bp-user-query.php

```diff
@@ -279,14 +279,16 @@ public function prepare_user_ids_query() {
 				$sql['select']   = "SELECT u.{$this->uid_name} as id FROM {$this->uid_table} u";
 				$sql['where'][]  = $wpdb->prepare( "u.component = %s AND u.type = 'last_activity'", buddypress()->members->id );
 
+				$online_default_time = apply_filters( 'bb_default_online_presence_time', bb_presence_interval() + 5 );
```

Comment:
extra span time should be a separate variable that should be changeable using filter.